config/settings.yml

```diff
@@ -1140,6 +1140,7 @@
       :event_streams_purge_interval: 1.day
       :evm_snapshot_delete_delay_for_job_not_found: 1.hour
       :evm_snapshot_interval: 1.hour
+      :infra_conversion_dispatcher_interval: 30.seconds
```

Comment:
I'm curious how you came up with that number.  Not sure if it's hardcoded elsewhere.

Reply:
It used to be run along with everything else every 15 seconds but that seemed like overkill for v2v since it is such a long process